data/src/main/java/com/microsoft/azure/kusto/data/exceptions/TriState.java

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.azure.kusto.data.exceptions;
+
+public enum TriState {
+    TRUE,
+    FALSE,
+    DONTKNOW
```

Comment:
DONT_KNOW or UNKNOWN

Reply:
+1 for UNKNOWN

Reply:
DONT_KNOW is from the tristate in our .Net